appcheck/firebase-appcheck-debug/src/main/java/com/google/firebase/appcheck/debug/internal/DebugAppCheckProvider.java

```diff
@@ -83,6 +83,16 @@ public DebugAppCheckProvider(
     this.debugSecretTask = Tasks.forResult(debugSecret);
   }
 
+  @VisibleForTesting
```

Comment:
it would be nice to have this not just visible for testing but for normal usage too :) right now distributing the apk manually i have to get a lot of peoples keys to be able to use vertex AI.